troubleshooting/csp.md

```diff
@@ -20,29 +20,33 @@ In general, a strict CSP can disable web app features, such as:
 * Inline JavaScript in `<script>` tags and legacy DOM attributes such as `onclick`. Blazor `@onclick` directives work as expected.
 * Inline CSS in `<style>` tags and `style` attributes. 
 * Fonts and images that are embedded in stylesheets with `data:` URIs.
-* Dynamic code evaluation via `eval()`.
+* Dynamic JavaScript code evaluation with `eval()`.
 
 ## Policy Configuration
 
 Telerik UI for Blazor components need the following exceptions to strict CSP. Some of them depend on the product version or product usage.
 
-* Allow inline styles to use component parameters such as `Width`, `Height`, `RowHeight`, `ItemHeight`, `Top`, `Left`, etc. In addition, some components rely on inline styles for their rich functionality and UX.
-* Allow `data` URIs for images that are embedded in the [CSS themes]({%slug general-information/themes%}). These images are used for styled checkboxes and radio buttons, Slider ticks, and others.
-* Allow `https://blazor.cdn.telerik.com` as a source when using [the Telerik CDN]({%slug getting-started/what-you-need%}#using-cdn) for styles or scripts.
-* *(up to version 4.4.0 only)* Allow `unsafe-eval` to use [Chart label templates]({%slug components/chart/label-template-format%}). These templates used to rely on `eval()`. Since version 4.5.0, the Chart labels support a different template mechanism, which doesn't require `unsafe-eval`.
-* *(up to version 4.6.0 only)* Allow `data` URIs for `font-src` to use [font icons]({%slug common-features-icons%}). Later versions use a separate file for the `WebComponentsIcons` icon font. This font file is referenced by the [font icon stylesheet]({%slug common-features-icons%}#font-icon-stylesheet).
+* Allow inline styles (`'unsafe-inline'`) to use component parameters such as `Width`, `Height`, `RowHeight`, `ItemHeight`, `Top`, `Left`, etc. In addition, some components rely on inline styles for their rich functionality and UX.
+* Allow data URIs (`data:`) for images that are embedded in the [CSS themes]({%slug general-information/themes%}). These images are used for styled checkboxes and radio buttons, Slider ticks, and others.
+* *(starting from version 6.0.0)* Allow script evaluation (`'unsafe-eval'`), which is required by the [Spreadsheet]({%slug spreadsheet-overview%}) for cell validation.
```

Comment:
The usual form is `starting with`. It seems we haven't included this in the style guide yet (I will fix this as soon as possible). Here's [another resource](https://english.stackexchange.com/questions/587351/starting-from-version-vs-starting-with-version-vs-beginning-with-version) on the subject in the meantime.